ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/impl/StreamedQueryResultImpl.java

```diff
@@ -127,6 +144,42 @@ private void handleErrorWhilePolling(final Throwable t) {
     log.error("Unexpected error while polling: " + t);
   }
 
+  @Override
+  public boolean hasContinuationToken() {
+    return !this.continuationToken.isPresent();
+  }
+
+  @Override
+  public Optional<String> getContinuationToken() {
+    return this.continuationToken;
+  }
+
+  @Override
+  public CompletableFuture<StreamedQueryResult> retry(
+      final int maxRetries) throws InterruptedException, ExecutionException {
+    long waitMs = 100;
+
+    requestProperties.put("request.ksql.query.push.start.offsets", continuationToken.get());
```

Comment:
this is redundant since we do this in streamQuery already, and it also uses the old config name which is wrong